app/views/reportings/_toolbar.html.erb

```diff
@@ -0,0 +1,28 @@
+<% type ||= :show %>
+<% actions ||= false %>
```

Comment:
I'd say you should add a verb here, e.g. `show_actions`. But that might be personal taste...